src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -162,12 +205,6 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
 
     Motion *solution       = lastGoalMotion_;
 
-    // \todo Make this variable unnecessary, or at least have it
-    // persist across solve runs
-    base::Cost bestCost    = opt_->infiniteCost();
-
-    bestCost_ = opt_->infiniteCost();
-
```

Comment:
Last summer I was trying to remove this as well but at the end I didn't. In my case I was using both for the CForest framework, but now I agree that bestCost can be simply removed. It would be nice if you can run the CForest demo/tests with these changes to be sure that nothing is broken.